life_expectancy/cleaning.py

```diff
@@ -0,0 +1,66 @@
+"""
+Assignment 1
+"""
+import argparse
+import numpy as np
+import pandas as pd
+
+def load_data():
```

Comment:
Can you add a return type hint?
```suggestion
def load_data() -> pd.DataFrame:
```